src/test/java/ru/svgogin/service/spark/web/SparkControllerTest.java

```diff
@@ -198,20 +198,184 @@ void deleteCompanyShouldDeleteEntities() throws Exception {
     // given
     when(sparkService.delete(any())).thenReturn(bankDto);
     //when
-    mockMvc.perform(MockMvcRequestBuilders.delete("/spark/companies/07725038124"))
+    mockMvc.perform(MockMvcRequestBuilders.delete("/spark/companies/7725038124"))
         .andDo(print())
         .andExpect(MockMvcResultMatchers.status().isOk());
     //then
-    verify(sparkService).delete(argThat(inn -> inn.equals("07725038124")));
+    verify(sparkService).delete(argThat(inn -> inn.equals("7725038124")));
   }
 
   @Test
   void deleteCompanyShouldReturn404ifNotExist() throws Exception {
     // given
     when(sparkService.delete(any())).thenThrow(NoSuchEntityException.class);
     //when
-    mockMvc.perform(MockMvcRequestBuilders.delete("/spark/companies/97725038124"))
+    mockMvc.perform(MockMvcRequestBuilders.delete("/spark/companies/9772503812"))
         .andDo(print())
         .andExpect(MockMvcResultMatchers.status().isNotFound());
   }
+
+  @Test
+  void getCompanyByInnShouldThrowException_WhenPathInnIsNotValid() throws Exception {
+    // given
+    // when
+    mockMvc.perform(MockMvcRequestBuilders.get("/spark/companies/77725038124"))
+        .andDo(print())
+        .andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.content().json(
+            "[{\n"
+            + "    \"code\": \"ERROR003\",\n"
+            + "    \"message\": \"getCompanyByInn.inn: Inn should contain 10 or 12 digits\"\n"
+            + "}]\n"
+            )
+        );
+  }
+
+  @Test
+  void saveCompanyShouldThrowException_WhenOgrnInBodyIsNotValid() throws Exception {
```

Comment:
```suggestion
  void saveCompanyShouldThrowException_WhenOgrnIsNotValid() throws Exception {
```

And also, you can use snake_case for tests:
```suggestion
  void save_company_should_throw_exception_when_ogrn_is_not_valid() throws Exception {
```

Reply:
Ok